ctapipe_io_lst/pointing.py

```diff
@@ -20,10 +21,15 @@
 
 
 class PointingSource(TelescopeComponent):
-    """Provides access to pointing information stored in LST drive reports."""
+    """Provides access to pointing information stored in LST drive reports.
+    """
     drive_report_path = traits.TelescopeParameter(
         trait=traits.Path(exists=True, directory_ok=False, allow_none=True),
-        help='Path to the LST drive report file',
+        help=(
+            'Path to the LST drive report file.'
+            ' This has to be the new format, where files are called'
+            ' DrivePosition_log_YYYYMMDD.txt'
```

Comment:
Maybe add that the corresponding bending model correction files are named `BendingModelCorrection_log_YYYMMDD.txt`, and they have to be placed in the same folder to use them